compose/ui/ui/src/desktopTest/kotlin/androidx/compose/ui/ComposeSceneTest.kt

```diff
@@ -307,7 +307,7 @@ class ComposeSceneTest {
 
         scene.sendPointerEvent(PointerEventType.Move, Offset(-1f, -1f))
         scene.sendPointerEvent(PointerEventType.Exit, Offset(-1f, -1f))
-        awaitNextRender()
+        skipRenders()
```

Comment:
Let's await 2 frames instead of 1000 (based of implementation of `skipRenders`). Or use something like `waitForIdle`.

Also, please add comment with link to the issue